Bottom: ccf1c3ae432b3ce1789185552df6b20b965cdd01
Top:    1f28a6ad315bc971074f895c2adedbcf96f77b94
Author: Parth Shah <parth@linux.ibm.com>
Date:   2020-02-07 14:42:10 +0530

Refresh of sched-move-available_idle_cpu

---

diff --git a/kernel/sched/fair.c b/kernel/sched/fair.c
index 36599b6155ea..8a72d6e929b3 100644
--- a/kernel/sched/fair.c
+++ b/kernel/sched/fair.c
@@ -5461,13 +5461,13 @@ static unsigned long capacity_of(int cpu)
  */
 int available_idle_cpu(int cpu)
 {
-       if (!idle_cpu(cpu))
-               return 0;
+	if (!idle_cpu(cpu))
+		return 0;
 
-       if (vcpu_is_preempted(cpu))
-               return 0;
+	if (vcpu_is_preempted(cpu))
+		return 0;
 
-       return 1;
+	return 1;
 }
 
 static void record_wakee(struct task_struct *p)
